godot/src/global.gd

```diff
@@ -122,13 +122,19 @@ func _ready():
 	get_tree().root.add_child.call_deferred(self.portable_experience_controller)
 	get_tree().root.add_child.call_deferred(self.testing_tools)
 	get_tree().root.add_child.call_deferred(self.metrics)
+	get_tree().root.add_child.call_deferred(self.network_inspector)
```

Comment:
can we avoid adding it to the root, if it is disabled :thinking: ?


and here (dcl_global.rs):
```
pub fn get_network_inspector_sender() -> Option<NetworkInspectorSender> {
```

Do we check if the node exists?

Reply:
the option is only if the Global doesn't exist (initial register of godot-rust)
if we don't treat it as node, we'd need to call the update anyway.